lite.auto.tfvars.example

```diff
@@ -0,0 +1,264 @@
+#-----------------------------------------------------------------------------
+
+# Name of the A record, of following dns zone, where the ecl watch ip is placed
+# This A record will be created and therefore should not exist in the following
+# dns zone.
+# Example entry: "my-product". This should be something project specific rather
+# than something generic.
+# Value type: string
+# Updateable: Y
+
+a_record_name="<eclwatch A record name>"
+
+#-----------------------------------------------------------------------------
+
+# Name of an existing dns zone.
+# Example entry: "hpcczone.us-hpccsystems-dev.azure.lnrsg.io"
+# Value type: string
+# Updateable: N
+
+aks_dns_zone_name="<dns zone name>"
+
+#-----------------------------------------------------------------------------
+
+# Name of the resource group of the above dns zone.
+# Example entry: "app-dns-prod-eastus2"
+# Value type: string
+# Updateable: N
+
+
+aks_dns_zone_resource_group_name="<dns zone resource group name>"
+
+#------------------------------------------------------------------------------
+
+# The version of HPCC Systems to install.
+# Only versions in nn.nn.nn format are supported.
```

Comment:
Minor: Isn't the string literal "latest" also supported?

Reply:
_Fixed_